roles/ipareplica/library/ipareplica_test.py

```diff
@@ -181,6 +181,32 @@
     type: bool
     default: no
     required: no
+  dot_forwarders:
+    description: List of DNS over TLS forwarders
+    type: list
+    elements: str
+    default: []
+    required: no
+  dns_over_tls:
+    description: Configure DNS over TLS
+    type: bool
+    default: no
+    required: no
+  dns_over_tls_cert:
+    description:
+      Certificate to use for DNS over TLS. If empty, a new
+      certificate will be requested from IPA CA
+    type: str
+    required: no
+  dns_over_tls_key:
+    description: Key for certificate specified in --dns-over-tls-cert
```

Comment:
Even if it is an "internal parameter", `--dns-over-tls-cert` is the CLI parameter and I'd prefer the role parameter `dns_over_tls_cert` to be used here.
(But I'm easily convinced otherwise, and this is more nitpicking than anything else.)